src/routes/_layout.svelte

```diff
@@ -1,11 +1,13 @@
 <script lang="ts">
-  import Navbar from "../components/sections/navbar.svelte";
-  import Footer from "../components/sections/footer.svelte";
+  import Navbar from "@/components/sections/navbar.svelte";
+  import Footer from "@/components/sections/footer.svelte";
+  import SkipNavbarContent from "@/components/utils/skip-navbar-content.svelte";
   export let segment: string;
   const pageTitle = segment === undefined ? "ACM at CSUF" : segment;
 </script>
 
 <Navbar segment="{segment}" />
+<SkipNavbarContent />
```

Comment:
Here is where the `SkipNavbarContent` component is implemented. It affects all pages of the site due to its placement in the [_special `_layout.svelte` file_](https://sapper.svelte.dev/docs#Layouts).